src/main/java/seedu/address/logic/commands/ListPrescriptionCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.ModelPrescription.PREDICATE_SHOW_ALL_PRESCRIPTIONS;
+
+import seedu.address.model.ModelPrescription;
+
+/**
+ * Lists all prescriptions stored to the user.
+ */
+public class ListPrescriptionCommand extends CommandPrescription {
+
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all prescriptions stored.\n"
```

Comment:
Commenting it here for now:

We will need to standardise cuz currently some no fullstop, some have. 
Lets standardise as every sentence in the result text box will have a fullstop. So do change/add to it as ure doing ur features :>>